web_translate_dialog/static/src/js/web_translate_dialog.js

```diff
@@ -216,6 +216,9 @@ var TranslateDialog = Dialog.extend({
                 if (code === self.view_language) {
                     _.each(text, function(value, key) {
                         var view_elem = self.view.$('input[name="'+ key + '"]');
```

Comment:
you could use `:input` to get the input no matter the type https://api.jquery.com/input-selector/